src/lib/y2storage/devicegraph.rb

```diff
@@ -119,11 +119,23 @@ def actiongraph(from: nil)
       Actiongraph.new(graph)
     end
 
+    # @return [Array<Dasd>]
+    def dasds
+      Dasd.all(self)
+    end
+
     # @return [Array<Disk>]
     def disks
       Disk.all(self)
     end
 
+    # All disk devices. This method is necessary because DASD devices
+    #   are not returned when asking for disks.
```

Comment:
The documentation of this method seems to be written taking into account the reader will have the implementation in front, so it's more focused in "why" that in "what". I would go for something more descriptive, since this is a very important method. Let me try an example:
```ruby
# All the devices that are usually treated like disks by YaST
#
# Currently this method returns an array including all the disks and all the DASDs.
# @see #disks
# @see #dasds
# @return [Array<Dasd, Disk>]
  def disk_devices
```
And then, I would add additional notes to the documentation of `disks` and `dasds`. Something like...
```ruby
# All the disks in the devicegraph
#
# @note Take into account that, based on the libstorage classes hierarchy, DASDs are not considered to be disks
# @see #disks_devices for a method providing the whole list of both disks and DASDs
# @return [Array<Disk>]
def disks
```